design_introspection-plugin/get_ports.cc

```diff
@@ -18,6 +18,7 @@
  *
  */
 #include "get_ports.h"
+#include <boost/algorithm/string.hpp>
```

Comment:
Do we have other uses of `boost` in `yosys-symbiflow-plugins`?  I don't think adding boost just for `trim` makes sense.

Reply:
We do not, but it's not something we are adding. Yosys has the boost libraries and we are building our plugins with Yosys's libraries. So we are not adding anything, we are just using something that Yosys uses already.

Reply:
I'd prefer we not add this dependency.  `trim` is a pretty small function, and I'd rather not add the dependency here.  If yosys drops `boost`, I'd rather not keep it just for the `trim` function.

Please implement a `trim` function, and add unit test or two.

Reply:
Done